app/models/task.rb

```diff
@@ -0,0 +1,3 @@
+class Task < ApplicationRecord
+  belongs_to :user
```

Comment:
What do you think about adding validations? It's a good practice, you can read more [here](http://rusrails.ru/active-record-validations#zachem-ispolzovat-validatsii).

```suggestion

  validates :name, :description, presence: true
```